pylint/checkers/classes/different_parameters.py

```diff
@@ -0,0 +1,121 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+"""Deal with parameters overriding in two methods."""
```

Comment:
The functions here should be part of `class_checker.py`. They aren't used anywhere else.